TalkAloud/TalkAloud/Model/AudioEngine.swift

```diff
@@ -46,6 +46,7 @@ class AudioEngine: NSObject {
     
     // Intializing audioRecorder here to make clear
     // when I'm initializing the audioRecorder and actually recording
+    // TODO: Change fileURL to use attribute url
```

Comment:
What do you mean with this TODO?

Reply:
I forgot I made this but I was thinking at first that I was going to have to use the url property from CoreData. I thought I deleted this maybe this was from that commit I went back to.